tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/model/Type.java

```diff
@@ -22,7 +22,15 @@
  */
 public enum  Type {
```

Comment:
Could you please add property `propertyFileName` for name of the properties file? and also use one property file for `ZIPKIN` and `APM`?